lib/drives/pages/drive_detail_page.dart

```diff
@@ -369,7 +387,9 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
 
   void _cancelDrive() async {
     await _drive?.cancel();
-    setState(() {});
+    setState(() {
```

Comment:
shouldn't the cancel method set cancelled true?

Reply:
yes sorry that should not have been in there. nice catch!